httpdocs/js/project.js

```diff
@@ -112,6 +112,9 @@ export default class Project extends User {
       (data ? data.version : this.findGetParameter('version'));
     const src = 'https://cyberbotics.com/wwi/' + version + '/WebotsView.js';
 
+    if (!data)
+      that._updateSimulationViewCount(url);
```

Comment:
Is it useful?
It seems like the returned data is not handled

Reply:
If `data` is set, it is either an animation or a scene, so here all we do is call an iterator to update the simulation view count for that particular url (simulation) in the database.